.github/vale/styles/Aiven/aiven_spelling.yml

```diff
@@ -0,0 +1,8 @@
+extends: spelling
+message: "'%s' seems to be a typo"
```

Comment:
how about "seems not to be a recognised word" ? It's not necessarily mis-typing

Reply:
Thanks, that's much better. I'd been staring at this so long I couldn't think of good phrases.
The pedant in me make me choose "does not seem to be a recognised word" - complain again if that's really too long...